docs/en_US/Release.rst

```diff
@@ -5,6 +5,71 @@
 Change Log
 ==========
 
+Release 2.1 - 3/10/2021
+-----------------------
+
+Major updates
+^^^^^^^^^^^^^
+
+Neural architecture search
+""""""""""""""""""""""""""
+
+* Improve NAS 2.0 (Retiarii) Framework (Improved Experimental)
```

Comment:
@QuanluZhang, please check the description of "Improved Experimental", thank you~